.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,29 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '40 6 * * 3'
```

Comment:
seems arbitrary, but sure

Reply:
Was Github's default when creating a CodeQL workflow :man_shrugging:

Reply:
So, yeah. Is arbitrary :P